custom/icds_reports/migrations/sql_templates/database_views/agg_awc_monthly.sql

```diff
@@ -85,7 +85,10 @@ CREATE VIEW agg_awc_monthly AS
         COALESCE("agg_awc"."cases_person_has_aadhaar_v2", 0) AS "cases_person_has_aadhaar_v2",
         COALESCE("agg_awc"."cases_person_beneficiary_v2", 0) AS "cases_person_beneficiary_v2",
         "agg_awc"."infantometer" AS "infantometer",
-        "agg_awc"."stadiometer" AS "stadiometer"
+        "agg_awc"."stadiometer" AS "stadiometer",
```

Comment:
hey, @mkangia can you also add these columns to its django model view. here https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/models/views.py#L365